packages/medusa/src/commands/utils/index.ts

```diff
@@ -0,0 +1,21 @@
+import { logger } from "@medusajs/framework"
+import { MedusaContainer } from "@medusajs/types"
+import { ContainerRegistrationKeys } from "@medusajs/utils"
+
+export async function ensureDbExists(container: MedusaContainer) {
+  const pgConnection = container.resolve(
```

Comment:
Q: if the database does not exists, the pg connection loader would have fail to create the pg_connection right? in that case, does this make sense and in that case should we do that before initializing the container in the sync links file?

Reply:
I think creating the connection (ie the knex instance) does not check with the database server unless you issue a query.

Reply:
you are right